### PR TITLE
named cloudformation resources

### PR DESCRIPTION
**Pull request by akash1810**
I'm going to be adding new sections for wowza resources, updating current cfn with logical naming to avoid confusion later on

### CODE REVIEW

**Comment by akash1810** [comment]
cc @guardian/multimedia

File: cfn/cfn.json [hunk 1]
```diff
@@ -227,114 +98,82 @@
...
+                            ],
+                            "Resource": "*"
+                        }, {
+                            "Effect": "Allow",
+                            "Action": [
+                                "s3:HeadObject",
+                                "s3:GetObject",
+                                "s3:GetObjectAcl"
+                            ],
+                            "Resource": "arn:aws:s3:::gnm-multimedia-rr-deployables/*"
```
Comment (gidsg) [inline cfn/cfn.json]: very minor but this isn't about getting github keys so probably should be moved into its own policy for clarity.

File: cfn/cfn.json [hunk 2]
```diff
@@ -347,14 +186,9 @@
...
                 "VpcId": "vpc-d5c5cab0"
-            },
-            "Metadata": {
-                "AWS::CloudFormation::Designer": {
-                    "id": "29d9f23a-24b8-4297-b96c-18e0465ac54f"
-                }
             }
         },
-        "EC2SG3IL8D": {
+        "LiveStreamLoadBalancerSecurityGroup": {
```
Comment (gidsg) [inline cfn/cfn.json]: From an infosec point of view I would suggest restricting this to guardian IP range.
i.e. `"CidrIp": "77.91.248.0/21"`

**Comment by gidsg** [comment]
👍 ❤️ (aside from minor comments)

File: cfn/cfn.json (hunk 1 above)
Comment (akash1810) [inline cfn/cfn.json]: good point, will make the change.

File: cfn/cfn.json (hunk 2 above)
Comment (akash1810) [inline cfn/cfn.json]: good call - we'll be putting [panda](https://github.com/guardian/pan-domain-authentication) in soon to open it up worldwide.

**Pull request closed without merging**